back-end/test/postUsers.js

```diff
@@ -15,6 +15,16 @@ describe('POST /tv_users', () => {
     stub.restore()
   })
 
+  after(() => {
```

Comment:
you're also going to want to add const mongoose = require('mongoose') after line 5.